pkg/identityserver/user_registry.go

```diff
@@ -469,6 +469,11 @@ func (is *IdentityServer) updateUser(ctx context.Context, req *ttnpb.UpdateUserR
 	}
 
 	err = is.store.Transact(ctx, func(ctx context.Context, st store.Store) (err error) {
+		err = isLastAdmin(ctx, st, req.User.Ids)
```

Comment:
I think we should do this check only if the field mask really contains `admin`.

Reply:
Agreed, also removed the `!req.user.admin` validation since if the field mask is present the change is implied